tests/openapi/test_responses.py

```diff
@@ -1,5 +1,7 @@
 from http import HTTPStatus
 
+import pytest
+from pydantic.main import BaseModel
```

Comment:
```suggestion
from pydantic import BaseModel
```

Reply:
Addressed in https://github.com/starlite-api/starlite/pull/478/commits/dc2b02e52214f4ef2be388ce4d5f114615bea6bb